packages/ffe-form-react/src/RadioSwitch.js

```diff
@@ -61,13 +61,13 @@ RadioSwitch.propTypes = {
     /** The label of the choice to the left */
     leftLabel: string.isRequired,
     /** The value of the choice to the left */
-    leftValue: oneOfType([bool, string]).isRequired,
+    leftValue: oneOfType([number, string]).isRequired,
```

Comment:
Not a fan of this tbh. for a component called a "switch", using a `bool` for the state/model value really is the only sensible choice, and React will make sure it ends up in the DOM as a DOMString because that is what we're using React for in the first place... abstracting *away* the DOM.

I don't agree that basing `propTypes` in React on what props are supposed to be in the DOM because the mental model React wants us to use is to not be aware there even is a DOM.

Reply:
I mean, we're already using `onChange` more or less everywhere even though React's `onChange` does not act like the DOM `change` event at all but more like the `input`.

We should not have to worry so much about the restraints put on us by the DOM.